services/github/github-contributors.service.js

```diff
@@ -42,7 +42,6 @@ module.exports = class GithubContributors extends LegacyService {
           message: '397',
           color: 'blue',
         },
-        keywords: ['GitHub', 'contributor'],
```

Comment:
More of a question as I'm not familiar with search, but does it do partial word matches? I.e. will `contributor` still be matched in search since the title contains `contributors`

Reply:
Yea. I think it matches on leading substrings:

<img width="835" alt="screen shot 2018-12-19 at 4 51 45 pm" src="https://user-images.githubusercontent.com/1487036/50250317-6c704280-03ae-11e9-9462-934b630eda3a.png">

Reply:
Nice!